File: TaskManager_56/src/main/java/issues/Issue.java
package issues;

import javax.xml.crypto.Data;
import java.util.Date;

public class Issue {
    public int id;
    public String title;
    public  String description;
    public  enum Priority{
        Low,Medium,High,Critical
    };

    public enum status{
        Open, InProgress, UnderReview, Resolved, Closed
    };
    public Date creationDate;
    public Date LastModified;
    public String tag;


    public Issue(int id){
        this.id=id;

    }


    public void create(){
        
    }

}

File: issue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace taskManager
{
    public enum priority
    {
        Low,
        Medium,
        High,
        Crtitical
    }

    public enum status
    {
        Open,
        Inprogress,
        UnderReview,
        Resolved,
        Closed
    }
    public class issue : ICloneable
    {
        public int id;
        public string title;
        public string description;
        public enum priority { Low, Medium, High, Crtitical }
        public enum status { Open, Inprogress, UnderReview, Resolved, Closed }
        public DateTime creation_date;
        public DateTime last_modified_date;
        public string assignedto { get; set; }
        List<string> tags = new List<string> { };
        List<object> Icomments = new List<object> { };

        public issue(string title, string description, Enum priority)
        {
            this.title = title;
            this.description = description;
        }

        public object clone
        {
           issue clonedissue = new issue();

         

            
        }
    }
}
